docs/standard/datetime/system-text-json-support.md

```diff
@@ -110,6 +113,28 @@ the converter successfully parses the data using <xref:System.DateTime.Parse(Sys
 
 [!code-csharp[example-showing-datetime-parse-as-fallback](~/samples/snippets/standard/datetime/json/datetime-converter-examples/example3/Program.cs)]
 
+### When writing with <xref:System.Text.Json.Utf8JsonWriter>
+
+If you want to write a custom <xref:System.DateTime> or <xref:System.DateTimeOffset> text representation with <xref:System.Text.Json.Utf8JsonWriter>,
+you can format your custom representation to a <xref:System.String>, `ReadOnlySpan<Byte>`, `ReadOnlySpan<Char>`, or <xref:System.Text.Json.JsonEncodedText>,
+then pass it to the corresponding <xref:System.Text.Json.Utf8JsonWriter.WriteStringValue> method.
+
+The following example shows how a custom <xref:System.DateTime> format can be created with <xref:System.DateTime.ToString(System.String,System.IFormatProvider)>,
+then written with the <xref:System.Text.Json.Utf8JsonWriter.WriteStringValue(System.String)> method:
+
+[!code-csharp[example-custom-writing-with-utf8jsonwriter](~/samples/snippets/standard/datetime/json/custom-writing-with-utf8jsonwriter.cs)]
+
+### When reading with <xref:System.Text.Json.Utf8JsonReader>
+
+If you want to read a custom <xref:System.DateTime> or <xref:System.DateTimeOffset> text representation with <xref:System.Text.Json.Utf8JsonReader>,
+you can get the value of the current JSON token as a <xref:System.String> using <xref:System.Text.Json.Utf8JsonReader.GetString>, then parse the value
+using custom logic.
+
+The following example shows how a custom <xref:System.DateTimeOffset> text representation can be retrieved using <xref:System.Text.Json.Utf8JsonReader.GetString>,
```

Comment:
Other than showcasing the exception path first, before the workaround, this looks good:
https://github.com/dotnet/samples/pull/1351/files#r319257875

Reply:
@ahsonkhan I've addressed this in https://github.com/dotnet/samples/pull/1412 and added a commit to this PR, please take a look.

Reply:
Re-looking at the whole doc, the latest changes and dotnet/samples#1412 may be redundant and hence we could undo them.